tests/lib/Repository/Helper/SchemaIdentifierExtractorTest.php

```diff
@@ -0,0 +1,45 @@
+<?php
+/**
+ * @copyright Copyright (C) Ibexa AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace Ibexa\Tests\Core\Repository\Helper;
+
+use Ibexa\Core\Repository\Helper\SchemaIdentifierExtractor;
+use PHPUnit\Framework\TestCase;
+
+class SchemaIdentifierExtractorTest extends TestCase
+{
+    public function testExtract()
+    {
+        $extractor = new SchemaIdentifierExtractor();
+
+        // Test with a valid schema string
+        $schemaString = '<field:username>|<relation:author>|<field:title>';
+        $expectedResult = [
+            'field' => ['username', 'title'],
+            'relation' => ['author']
+        ];
+        $this->assertEquals($expectedResult, $extractor->extract($schemaString));
```

Comment:
assertEquals is a static method and should be called statically:
```suggestion
        self::assertEquals($expectedResult, $extractor->extract($schemaString));
```